velox/dwio/parquet/reader/PageReader.cpp

```diff
@@ -188,6 +188,20 @@ const char* FOLLY_NONNULL PageReader::uncompressData(
           (ret == Z_OK),
           "zlib inflateInit failed: {}",
           stream.msg ? stream.msg : "");
+
+      struct InflateEndGuard {
+        explicit InflateEndGuard(z_stream* stream) : stream_(stream) {}
```

Comment:
```
VELOX_CHECK_NOTNULL(stream_)
```

Reply:
This is just a local struct and we are guaranteed to pass a non-null pointer